requirements.txt

```diff
@@ -69,7 +69,7 @@ python-rapidjson==0.8.0
 redis==2.10.6
 redis-py-cluster==1.3.5
 semaphore>=0.4.64,<0.5.0
-sentry-sdk==0.13.1
+sentry-sdk>=0.13.5
```

Comment:
This should have _some_ upper bound (probably minor release version, but I'm not familiar with how strictly we've stuck to semver convention for these more experimental features) to avoid surprises if we're lagging behind on backwards incompatible SDK changes.

Reply:
I copied what we do on sentry. But yeah, makes sense. Will add the upper bound